src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -1397,6 +1397,11 @@ bool clang_c_convertert::get_builtin_type(
     c_type = "__uint128";
     break;
 
+  case clang::BuiltinType::NullPtr:
+    new_type = pointer_type();
+    c_type = "uintptr_t";
```

Comment:
Why is the c_type uintptr_t?

Reply:
Because it's not supposed to be a pointer type, but have the same size as a pointer.